### PR TITLE
Fix input_double

### PR DESCRIPTION
**Pull request by billsix**
Input double was using input_integer

### CODE REVIEW

File: imgui/cimgui.pxd [hunk 1]
```diff
@@ -864,9 +864,9 @@ cdef extern from "imgui.h" namespace "ImGui":
             ImGuiInputTextFlags extra_flags
     ) except +  # Widgets: Sliders (tip: ctrl+click on a slider to input with keyboard.
     bool InputDouble(  # ✓
-            const char* label, double v,
+            const char* label, double* v,
```
Comment (swistakm) [inline imgui/cimgui.pxd:867]: Hmm the signature in `imgui.h` is:
```c++
IMGUI_API bool          InputDouble(const char* label, double* v, double step = 0.0f, double step_fast = 0.0f, const char* format = "%.6f", ImGuiInputTextFlags extra_flags = 0);
```
And we're passing a pointer here so I think this should stay as `double* v`

Comment (billsix) [reply imgui/cimgui.pxd:867]: I don't understand.  My commit makes it pass the pointer to the double.

Comment (swistakm) [reply imgui/cimgui.pxd:867]: 🤦  nevermind

I'm just tired an misread the diff :D

**Comment by swistakm** [comment]
Great!

**Comment by swistakm** [comment]
Released as `1.3.0`:
- [release notes](https://github.com/swistakm/pyimgui/releases/tag/1.3.0)
- [available distributions](https://pypi.org/project/imgui/1.3.0/#files)

**Pull request merged**